### ISSUE TITLE
Cannot bypass prompt with empty string answer

### ISSUE THREAD

**Issue by Pike** [plopjs/plop#302]
I'm trying to create bypass arguments for one of my plops like so

```
plop-wrapper generator-name -- --param=""
```

but that doesn't mute the prompt.

AFAICT, the culprit is 

https://github.com/plopjs/plop/blob/3fe7308b21ae9dfcebcbea8a59af252b20e6a0c5/src/bypass.js#L55

which checks for truthy, not for existence?

**Comment by crutchcorn** [comment]
I think you're 100% right. Do you wanna tackle a PR for this (with E2E) or would you prefer I do so tomorrow?

Quick aside: I deeply appreciate your research and in-depth bug reporting. Thank you so much

**Comment by Pike** [comment]
I gave it a shot, but I failed on the E2E testing this time around. Either the test just passed regardless of my code change, or hang regardless.

This is the fix I tested:

```diff
diff --git a/src/bypass.js b/src/bypass.js
index b500f64..a63578c 100644
--- a/src/bypass.js
+++ b/src/bypass.js
@@ -52,7 +52,7 @@ function combineBypassData(generator, bypassArr, plopArgV) {
       process.exit(1);
     }
     namedBypassArr = promptNames.map((name) =>
-      plopArgV[name] ? plopArgV[name] : undefined
+      plopArgV[name] !== undefined ? plopArgV[name] : undefined
     );
   }
 ```

**Comment by crutchcorn** [comment]
I was able to create a failing test that, sure enough, your patch fixed! Thanks @Pike 

Should be fixed as-of 3.0.4:

https://github.com/plopjs/plop/releases/tag/v3.0.4

**Comment by Pike** [comment]
Thanks
